### PR TITLE
fix: only use mpme and ntmpi in test config. fixes #302

### LINKED ISSUE #302: npme and ntmpi mdrun options
https://github.com/hits-mbm-dev/kimmdy/blob/1fc410a706bc8eb0c0a5ee0f972a175dc2a243cb/src/kimmdy/runmanager.py#L392

Does this line have any reason to exist, @KRiedmiller @jmbuhr ? I would like to specify those through the config file.

### PR DESCRIPTION
**Pull request by jmbuhr**

### CODE REVIEW

**Comment by KRiedmiller** [comment]
See #302   
Also, `-nt 8` seems bad, my machine and gh actions do not have 8 cores, this slows the simulation down

**Comment by jmbuhr** [comment]
How many cores do we have on gh? We can always go lower for the tests (not higher, then the domain decomposition fails)

**Comment by KRiedmiller** [comment]
I don't think we need to specify, gromacs should automatically use all available ones I think..? 2 cores on github

If domain decomposition can be an issue, can't we fix this with arguments?

**Comment by jmbuhr** [comment]
We need to specify, because by default gromacs tries to use as many cores as it can find. This fails for small systems on beefy machines (so it's a local WS testing problem, not a gh actions problem)

**Comment by KRiedmiller** [comment]
But doesn't it fail because of domain decomposition? We can specify `-dd 1 1 1` and domain decomposition not scale with the machine it runs on.  
Works for ala, need still to check for the other examples.

**Comment by jmbuhr** [comment]
True, that is also an option. Though for the small systems we may want to specify `nt` anyways, because too many fast cores can make it run slower again...

**Comment by KRiedmiller** [comment]
I'd prefer `-d`, but it's just about performance at that point, so we can keep `-nt`  
What is `-dbl`about? Neither me nor the gromacs search field did find this flag.

**Comment by jmbuhr** [comment]
oh, `dbl` is a typo that should be `dlb` (dynamic load balancing)

File: src/kimmdy/kimmdy-yaml-schema.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
       "default": "gmx"
     },
     "gmx_mdrun_flags": {
-      "description": "Flags passed to gmx mdrun. e.g. `-nt 8 -maxh 24`",
+      "description": "Flags passed to gmx mdrun. Default `-maxh 24 -dbl yes`",
```
Comment (KRiedmiller) [inline src/kimmdy/kimmdy-yaml-schema.json:105]:
```suggestion
      "description": "Flags passed to gmx mdrun. Default `-maxh 24 -dlb yes`",
```

**Review by KRiedmiller — APPROVED** [review]

**Pull request merged**